### DISCUSSION TITLE
Django-comments-xtd

### DISCUSSION THREAD

**Discussion by odescopi** [danirus/django-comments-xtd#263] [Q&A]
Good day dan.. please my project has a users app so I want only authenticated users to comment but it seems your login for unauthenticated users redirects to to Django admin login page (reply,likes, dislikes,flag) instead of a user login page I created..how do I reset it to my user login page..
Also it seems ur flagging doesn't render on the same page like reply,likes..it opens another page could it be possible?
Show quoted text
Hi @odescopi, please, ask this kind of questions in Discussion.
Thank you!

**Comment by odescopi** [comment]
Thanks guys I've figured it out from the answer of a previous similar comment here..

**Comment by ogurec-ogurec** [comment]
@odescopi I didn't quite understand the question, but if you have a problem with redirecting to the login page, then I had the same gap. The solution for [discussions is below](https://github.com/danirus/django-comments-xtd/discussions/247)


Fixing this line in the file helped me:

add setting.py: 

```
LOGIN_URL = '/login/'
LOGIN_REDIRECT_URL = '/login/'

```
and add files: 

`\Lib\site-packages\django_comments_xtd\api\frontend.py: 114:`

`d ['login_url']="/login/"`

**Reply by odescopi** [reply]
@ogurec-ogurec  i have been having issues loading the comments..i get this internal server error 500 ..in commentbox.jsx 173  in the console,after sending a comment the platform freezes and does not reveal the comments sent..please kindly check if u can assist me..iv successfully run the tutorial project and my project but yet the same error keep occurring..below is a screenshot
![tutorerror](https://user-images.githubusercontent.com/51009927/112995908-8fa70200-9120-11eb-8b1a-4428554ad1a3.PNG)
![ann](https://user-images.githubusercontent.com/51009927/112996208-d268da00-9120-11eb-851f-ff8bce83b9c8.PNG)
![ann2](https://user-images.githubusercontent.com/51009927/112996283-e44a7d00-9120-11eb-9743-4cb0f6b73bea.PNG)
